devicetypes/smartthings/zigbee-sound-sensor/zigbee-sound-sensor.groovy

```diff
@@ -0,0 +1,173 @@
+/**
+ *  Zigbee Sound Sensor
+ *
+ *  Copyright 2018 Samsung SRPOL
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
+
+metadata {
+    definition(name: "Zigbee Sound Sensor", namespace: "smartthings", author: "Samsung SRPOL") {
```

Comment:
you could explicitly reference the ui-metadata created for the Z-Wave Sound Sensor here, since they will be identical.

`mnmn: "SmartThings", vid: "SmartThings-smartthings-Z-Wave_Sound_Sensor"`